test/caesar_test.rb

```diff
@@ -1,2 +1,34 @@
-# YOUR TESTS GOES HERE
-# I PITY THE FOOL WHO DOESNT WRITE TESTS
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/caesar.rb'
+require 'pry'
+
+class CaesarTest < Minitest::Test
+
+  def setup
+    @cipher = Caesar.new
+  end
+
+  def test_it_exists
+    assert_instance_of Caesar, @cipher
+  end
+
+#   Iteration 0
+# Translate English to ciphertext
+#
+# lowercase letters
+#   $ caesar = Caesar.new
+#   => #<Caesar:0x007fa1ab98cac0>
+#   $ caesar.eng_to_cipher("the quick brown fox jumps over the lazy dog", 3)
+#   => "qeb nrfzh yoltk clu grkmp lsbo qeb ixwv ald"
+
+  def test_it_ordinates
+    assert_equal [116, 101, 115, 116], @cipher.eng_to_cipher("test", 0)
```

Comment:
From the spec, we can see that eng_to_cipher should return the encrypted text. the array of shifted values is a good start, just put that into a helper method instead of the eng_to_cipher method. For example, `rotated_codes` or `rotate_chars`